primerj/src/main/java/org/primer/primerj/core/TxBuilder.java

```diff
@@ -486,8 +492,8 @@ public Tx buildBCCTx(Address address, String changeAddress, List<Out> unspendOut
         }
 
         int size = TxBuilder.estimationTxSize(outs.size(), scriptPubKey, tx.getOuts(), address.isCompressed());
-        if (size > 1000) {
-            fees = (size / 1000 + 1) * fees;
+        if (size > 0) {
+            fees = size * (fees / 1000);
         }
```

Comment:
buildBCCTx - this is for Bitcoin Cash (a fork of Bitcoin) I believe, we don't have it with primer, so maybe no need to change this function